tests/go-validation/Dockerfile.v1api

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16
```

Comment:
Are minor golang versions usually stable or do we prefer to stay in an specific minor otherwise?

Reply:
It depends. 1.16 is the latest most stable version right now. 1.17 is currently unstable. Stable releases is something we should keep an eye on, in case we want to upgrade. For example versions after 1.13 only support module dependency while versions before would use a vendor file for dependencies.

Reply:
Makes sense.